encord/objects/ontology_object_instance.py

```diff
@@ -136,6 +144,22 @@ def _last_frame(self) -> Union[int, float]:
         else:
             return self._parent.number_of_frames
 
+    @property
+    def range_list(self) -> Ranges:
+        if self._range_only:
+            return self._range_manager.get_ranges()
+        else:
+            raise LabelRowError(
+                "No ranges available for this classification instance."
+                "Please ensure the classification instance was created with "
+                "the range_only property to True. "
+                "You can do ClassificationInstance(range_only=True) or "
+                "Classification.create_instance(range_only=True) to achieve this."
+            )
+
+    def is_range_only(self) -> bool:
```

Comment:
Could also be a property?

Reply:
keeping consistent with classification interface.